src/contracts/facilitators/gsm/OwnableFacilitator.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {VersionedInitializable} from '@aave/core-v3/contracts/protocol/libraries/aave-upgradeability/VersionedInitializable.sol';
+import {IOwnableFacilitator} from 'src/contracts/facilitators/gsm/interfaces/IOwnableFacilitator.sol';
+import {IGhoToken} from 'src/contracts/gho/interfaces/IGhoToken.sol';
+
+/**
+ * @title OwnableFacilitator
+ * @author Aave/TokenLogic
+ * @notice GHO Facilitator used to directly mint GHO to a given address.
+ */
+contract OwnableFacilitator is Ownable, VersionedInitializable, IOwnableFacilitator {
```

Comment:
probably no need to make it upgradeable, as it's super-simple and easy to replace at GhoToken level. wdyt

Reply:
yeah, i didn't want to make it upgradeable to be honest but was just following what i had seen the most heh.